### PR TITLE
chore: broken links cleanup

### PR DESCRIPTION
**Pull request by fladson**
This is an attempt to fix some broken links.

This was done using a broken URL scanner. What I couldn't resolve automatically I added a `TODO` to be later discussed with the team, so please jump in if you know where the new link is or if we can edit/remove it because it's now deprecated.

### CODE REVIEW

File: playbooks/cloudflare.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ description: Working with Cloudflare
 
 Cloudflare is configured via Terraform in the [Infrastructure repo](https://github.com/artsy/infrastructure/blob/master/terraform/staging/cloudflare.tf)
 
-See [Enabling DDos Protection for a Subdomain via Cloudflare](https://github.com/artsy/potential/wiki/Platform-FAQ#enabling-ddos-protection-for-a-subdomain-via-cloudflare) for details on Cloudflare configuration
+See [Enabling DDos Protection for a Subdomain via Cloudflare](#TODO) for details on Cloudflare configuration
```
Comment (mdole) [inline playbooks/cloudflare.md:12]:
```suggestion
See [Enabling DDos Protection for a Subdomain via Cloudflare](https://www.notion.so/artsy/Cloudflare-80c2c0ed608644199e3eeb9da36f3b94#b8f87c4885374c019932f29b0ec318cb) for details on Cloudflare configuration
```

File: playbooks/kubernetes.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ We use [Datadog](https://app.datadoghq.com/) for cluster monitoring and alerting
 - [Container Overview](https://app.datadoghq.com/containers?columns=container_name,container_cpu,container_memory,container_net_sent_bps,container_net_rcvd_bps,container_status,container_started&options=normalizeCPU&sort=container_memory,DESC)
 
 ### Allocating Kubernetes resources for apps.
-Kubernetes allows you to allocate CPU/Memory for apps. Here are some [guidelines](app-resource-tuning.md) on how to do it.
+Kubernetes allows you to allocate CPU/Memory for apps. Here are some [guidelines](#TODO) on how to do it.
```
Comment (mdole) [inline playbooks/kubernetes.md:33]:
```suggestion
Kubernetes allows you to allocate CPU/Memory for apps. Here are some [guidelines](https://www.notion.so/artsy/Guidelines-on-tuning-an-app-s-CPU-and-memory-consumption-in-a-Kubernetes-cluster-797977be895643af84015a7d4b60a5dc) on how to do it.
```

File: playbooks/kubernetes.md [hunk 3]
```diff
@@ -109,7 +109,7 @@ To evict all running pods from a node before terminating it, use the following p
 
 5) Wait for autoscaling to automatically launch a new instance in place of the terminated one.
 
-The command `./manage.py rotate_cluster_nodes` [in the Substance repo](https://github.com/artsy/substance#rotate-cluster-nodes-drain-termainate-existing-nodes-and-relace-with-new-ones) automates this procedure.
+The command `./manage.py rotate_cluster_nodes` [in the Substance repo](#TODO) automates this procedure.
```
Comment (mdole) [inline playbooks/kubernetes.md:112]:
```suggestion
The command `python manage.py replace_cluster_nodes {cluster_directory}` [in the Substance repo](https://github.com/artsy/substance#replace-cluster-nodes-drain-termainate-existing-nodes-and-relace-with-new-ones) automates this procedure.
```

**Review by mdole — APPROVED** [review]
good thinking @fladson! tackled a few of the TODOs, will keep poking at it if I have time later :) or feel free to hit me up if you're working on locating where some of these resources moved to and want to pair

File: resources/mobile/README.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ There are three distinct ways of developing iOS software at Artsy:
 - Writing Objective-C and Swift in [Eigen](https://github.com/artsy/eigen) (setup instructions in Readme),
   sometimes importing React Native components _from_ Emission (as Objective-C `ARComponentController` subclasses).
 - Developing React Native _and_ Objective-C and Swift using Emission and Eigen linked together, at the same time.
-  [Instructions are in the Eigen docs folder](https://github.com/artsy/eigen/blob/master/docs/using_dev_emission.md)
```
Comment (pvinis) [inline resources/mobile/README.md:31]: wow these are out of date!

I'll bring it up in the mobile practice.

Comment (fladson) [reply resources/mobile/README.md:31]: I added a link to eigen docs in the meanwhile to remove the 404 redirects.

**Comment by admbtlr** [comment]
Thanks @fladson !

**Pull request merged**